pkg/clusteragent/clusterchecks/handler_test.go

```diff
@@ -163,13 +163,13 @@ func TestHandlerRun(t *testing.T) {
 		h.Run(ctx)
 		runReturned <- struct{}{}
 	}()
-	testutil.AssertTrueBeforeTimeout(t, 10*time.Millisecond, 1*time.Second, func() bool {
+	testutil.AssertTrueBeforeTimeout(t, 10*time.Millisecond, 2*time.Second, func() bool {
```

Comment:
I think it'd be good to extract the timeout to a constant so if we need to change it, we just have to update it in one place.